test/admin/sync.test.js

```diff
@@ -225,38 +225,25 @@ describe('Sync', () => {
       expect(deployResult.status).toBe('created');
     });
 
-
-    it('can use the allow_destructive_changes param to do a destructive rule create', async() => {
+    it('can use do a create a rule with invalid sync schema', async() => {
```

Comment:
Not sure if this was intentional, but the phrasing of the test name is kinda awkward.

Maybe we can change it to "can create a rule with invalid sync schema", or if it was intentional, add quotes to change it to "can do a 'Create a rule' with invalid sync schema"

Reply:
Ah man, yeah this was a typo. Thanks!